### ISSUE TITLE
I get too many support mails related to the system-tray so I only close the app after 4 seconds during the first run when the tutorial prompt is shown.

### ISSUE THREAD

**Issue by churusaa** [rocksdanister/lively#1666]
I get too many support mails related to the system-tray so I only close the app after 4 seconds during the first run when the tutorial prompt is shown.

Don't see it as an issue, the application can still be terminated through taskmanager if required.

_Originally posted by @rocksdanister in https://github.com/rocksdanister/lively/issues/1665#issuecomment-1501216434_
            

If it is a frequently cited issue by users, as you say yourself, addressing it may merit further consideration. Perhaps it isn't a problem for you personally, but you are not the only user of this program, no? If you can point me to the misbehaving prompt, I can take a look myself, or I can just pull it when I get some free time and can get my eyes adjusted to the light in here.

You have a good day, now.

**Comment by rocksdanister** [comment]
I think you misunderstood/I was not being clear, in the past I used to show a similar dialog but without the wait time - what happened was people just close too quickly without reading.. and send me support emails asking how to quit/re-open Lively (both can be done from tray-menu.)

After the 4s delay was added the emails reduced drastically.. I still get some occasionally but it definitely helped a lot.

Lively is designed for the average user and I do take into account all the users.. for example wallpapers will close even if you terminate through taskmanager instead of closing safely.

System-tray and how to remove (some delete the desktop shortcut or install folder and ask how to remove https://github.com/rocksdanister/lively/discussions/251) are the frequent support questions I get, for system-tray the dialog is shown and for remove the uninstall script will kill Lively process before proceeding.
 
The ALT + F4 is not intentional, I am just handling the Window Close event (x button) which is what Alt + F4 is also sending I guess
https://github.com/rocksdanister/lively/blob/c27d2d04e9d4e921c83ba74465e0869402e4fc83/src/Lively/Lively.UI.WinUI/Views/MainWindow.xaml.cs#L407-L437

PS: You don't have to create new issues for the same problem, just comment on the closed one.

**Comment by churusaa** [comment]
Thanks! I'm still not used to the comment system on here. Mostly I've just used github as a repository for my own projects, but when I saw my kids installed a program on my PC that was also being maintained on here, I was so glad for the chance to help out.

I'll take a look at what you have there and do some digging to see if I can find a way to preserve this intended behavior while still allowing alt+f4, and I'll get back to you with however much I can figure out.

Peace! 😊
